docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  mongodb:
+    image: mongo:3
+
+  redis:
+    image: redis:latest
+
+  http:
+    image: nginx:latest
+
+  dynamodb:
+    image: deangiberson/aws-dynamodb-local:latest
+
+  await:
+    image: saltside/await
+    build: .
+    environment:
+      DYNAMO_DB_URL: http://dynamodb:8000
+      AWS_ACCESS_KEY_ID : "test"
```

Comment:
IMO we should remove these aws key variables and pass them to the test. This way we ensure the command passes `$@` along to `aws`.